addons/weapononback/XEH_postInit.sqf

```diff
@@ -0,0 +1,39 @@
+#include "script_component.hpp"
+
+["weapon", FUNC(onWeaponChange)] call CBA_fnc_addPlayerEventHandler;
+
+["unit", {
+    params ["_newUnit", "_oldUnit"];
+
+    private _inventoryEHs = _oldUnit getVariable [QGVAR(inventoryEHs), []];
+    { _oldUnit removeEventHandler _x } forEach _inventoryEHs;
+
+    _inventoryEHs = [
+        _newUnit addEventHandler ["InventoryOpened", {
+            params ["", "_fst", "_snd"];
```

Comment:
```suggestion
            params ["", "_first", "_second"];
```

Reply:
I don't think that this isn't clear for an average reader.

Reply:
it wasn't clear enough for me. Also consistency.